glib-macros/src/gboxed_shared_derive.rs

```diff
@@ -178,5 +178,20 @@ pub fn impl_gshared_boxed(input: &syn::DeriveInput) -> proc_macro2::TokenStream
         #impl_to_value_optional
 
         #impl_from_value
+
+        impl #crate_ident::translate::FromGlibPtrNone<*const #refcounted_type_prefix::InnerType> for #name {
```

Comment:
For completeness this should probably also be done for
 - `ToGlibPtr`
 - the non-shared `GBoxed` (`ToGlibPtr`, `FromGlibPtrXXX`, `ToGlibPtrMut`)

Reply:
Hehe... yeah, I saw that one coming, but I was not totally sure how to do it for GBoxed, so I went ahead and submitted the pr :)

Reply:
The same way how it works in `FromValue` / `ToValue` :) What problem did you run into?

Reply:
mind you I have not tried very hard... but the first question I run into is what types to use in the ffi... for Shared I can used the `InnerType` associated type

Reply:
The type itself, as that's what is also stored as pointer value in the `GValue`